### ISSUE TITLE
Accessibility: Fix MSSQL config page fastpass issues

### ISSUE THREAD

**Issue by kaydelaney** [grafana/grafana#42717]

**Comment by codeincarnate** [comment]
Closing as this should be fixed with #43982
